src/framework/shortcuts/qml/MuseScore/Shortcuts/EditShortcutDialog.qml

```diff
@@ -35,6 +35,7 @@ Dialog {
     function startEdit(shortcut, allShortcuts) {
         open()
         model.load(shortcut, allShortcuts)
+        model.currentShortcut = shortcut
```

Comment:
You could remove this line and the `setCurrentShortcut` method in C++ (i.e. make `currentShortcut` a readonly property), and instead put the contents of `setCurrentShortcut` in the `load` method.